### ISSUE TITLE
Handle ENVI headers for PCI images

### ISSUE THREAD

**Issue by tboggs** [spectralpython/spectral#74]
ENVI headers with `file type = PCI` require special read procedures to access the data in the image file (see issue #72). Should either add support for reading these images or raise an exception or warning for PCI images.

**Comment by pythonic2020** [comment]
Has this enhancement been implemented?  I tried reading a _.pix PCI cube with a _.pix.hdr file, and the image is all distorted when I view it.  The image displays normally in ENVI.  Image below is from viewing a 1000x1000 subset of the imported image using spectral imshow.

![Figure 6](https://user-images.githubusercontent.com/61034042/99883754-b25f6200-2be6-11eb-9552-b676ff91c595.png)

**Comment by tboggs** [comment]
No, it has not been implemented. If I had access to the `pci_read_spatial` and `pci_read_spectral` routines referenced in issue #72, I could probably implement it. In the mean time, you can likely get around it by adjusted the `samples` parameter in the header file to account for the additional data being appended to each line.

**Comment by pythonic2020** [comment]
I have the `pci_read_spatial `and `pci_read_spectral` in my header also.  I'll look into changing the samples param.  Thank you!!!!  :-)
